src/materials/icons/layout.pug

```diff
@@ -0,0 +1,32 @@
+//- inspired by https://github.com/jonathantneal/svg4everybody#readability-and-accessibility
+
+//- Example: should read "The title"
```

Comment:
Good documenting it here.
We'll move that later to the icons pages on the styleguide but it's great it's already here for now 👍